projects/kubernetes-csi/external-provisioner/Makefile

```diff
@@ -30,6 +30,8 @@ IMAGE_REPO?=$(AWS_ACCOUNT_ID).dkr.ecr.$(AWS_REGION).amazonaws.com
 IMAGE_NAME?=kubernetes-csi/external-provisioner
 IMAGE_TAG?=$(GIT_TAG)-eks-${RELEASE_BRANCH}-${RELEASE}
 IMAGE?=$(IMAGE_REPO)/$(IMAGE_NAME):$(IMAGE_TAG)
+GOPROXY_DNS?=direct
+export GOPROXY=$(GOPROXY_DNS)
```

Comment:
Why do you need to variables? Can you not just do
```Makefile
export GOPROXY?=direct
```

Same for below

Reply:
I tested your the suggested changes. It does not set the proxy without the variable.